deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql

```diff
@@ -45,7 +45,7 @@ MERGE(b:Badge {id: badge._id["$oid"]})
 ON CREATE SET
 b.key       = badge.key,
 b.type      = badge.type,
-b.icon      = badge.image.path,
+b.icon      = replace(badge.image.path, 'https://api-alpha.human-connection.org', ''),
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-14T19:07:45Z" title="Friday, June 14th 2019, 9:07:45 pm +02:00">Jun 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -45,7 +45,7 @@ MERGE(b:Badge {id: badge._id["$oid"]})
 ON CREATE SET
 b.key       = badge.key,
 b.type      = badge.type,
-b.icon      = badge.image.path,
+b.icon      = substring(badge.image.path, 38),
```


@ulfgebhardt I find it quite dangerous, how do you know that it's always the same first part of the URL?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-14T19:10:53Z" title="Friday, June 14th 2019, 9:10:53 pm +02:00">Jun 14, 2019</time>_

---


Use [replace](https://neo4j.com/docs/cypher-manual/current/functions/string/#functions-replace).